.circleci/config.yml

```diff
@@ -659,8 +659,11 @@ jobs:
             cp -a "$OUTPUT" /tmp/generated-docs
       - *save-cache-podmaster
       - *save-cache-gems
+      - run:
+          name: Compress Artifacts
+          command: tar -cvzf /tmp/docs.tar.gz /tmp/generated-docs
       - store_artifacts:
-          path: /tmp/generated-docs
+          path: /tmp/docs.tar.gz
```

Comment:
Used to take ~3m to upload each file separately, now zipping + uploading of 1 archive takes 5 sec.